### PR TITLE
jdk-17 April 2023 PSU Installers

### PR DESCRIPTION
**Pull request by andrew-m-leonard**

### CODE REVIEW

File: linux/jdk/redhat/src/main/packaging/temurin/17/temurin-17-jdk.spec [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+%global upstream_version 17.0.7+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 17.0.1.0.1___17.0.1.0+12
 #  17.0.1.0.0___12 == 17.0.1.0.0+12
-%global spec_version 17.0.6.0.0.10
-%global spec_release 3
+%global spec_version 17.0.7.0.0.7
+%global spec_release 4
```
Comment (steelhead31) [inline linux/jdk/redhat/src/main/packaging/temurin/17/temurin-17-jdk.spec:8]: This should be 1 as its the first spec version of 17.0.7

File: linux/jdk/redhat/src/main/packaging/temurin/17/temurin-17-jdk.spec [hunk 2]
```diff
@@ -261,6 +261,8 @@ fi
 /usr/lib/tmpfiles.d/%{name}.conf
 
 %changelog
+* Wed Apr 26 2023 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 17.0.7.0.0.7-4.adopt0
+- Eclipse Temurin 17.0.7+7 release 4.
```
Comment (steelhead31) [inline linux/jdk/redhat/src/main/packaging/temurin/17/temurin-17-jdk.spec:265]: Again, this should be release 1.

File: linux/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+%global upstream_version 17.0.7+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 17.0.1.0.1___17.0.1.0+12
 #  17.0.3.0.0___7 == 17.0.3.0.0+7
-%global spec_version 17.0.6.0.0.10
-%global spec_release 2
+%global spec_version 17.0.7.0.0.7
+%global spec_release 3
```
Comment (steelhead31) [inline linux/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec:8]: spec release 1 as this is the first version of this package

File: linux/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec [hunk 4]
```diff
@@ -251,6 +251,8 @@ fi
 %{prefix}
 
 %changelog
+* Wed Apr 26 2023 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 17.0.7.0.0.7-3.adopt0
+- Eclipse Temurin 17.0.7+7 release 3.
```
Comment (steelhead31) [inline linux/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec:255]: release 1.

File: linux/jre/redhat/src/main/packaging/temurin/17/temurin-17-jre.spec [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+%global upstream_version 17.0.7+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 17.0.1.0.1___17.0.1.0+12
 #  17.0.1.0.0___12 == 17.0.1.0.0+12
-%global spec_version 17.0.6.0.0.10
-%global spec_release 3
+%global spec_version 17.0.7.0.0.7
+%global spec_release 4
```
Comment (steelhead31) [inline linux/jre/redhat/src/main/packaging/temurin/17/temurin-17-jre.spec:8]: spec release 1

File: linux/jre/redhat/src/main/packaging/temurin/17/temurin-17-jre.spec [hunk 6]
```diff
@@ -186,6 +186,8 @@ fi
 /usr/lib/tmpfiles.d/%{name}.conf
 
 %changelog
+* Wed Apr 26 2023 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 17.0.7.0.0.7-4.adopt0
+- Eclipse Temurin JRE 17.0.7+7 release 4.
```
Comment (steelhead31) [inline linux/jre/redhat/src/main/packaging/temurin/17/temurin-17-jre.spec:190]: release 1.

File: linux/jre/suse/src/main/packaging/temurin/17/temurin-17-jre.spec [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
+%global upstream_version 17.0.7+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 17.0.1.0.1___17.0.1.0+12
 #  17.0.1.0.0___12 == 17.0.1.0.0+12
-%global spec_version 17.0.6.0.0.10
-%global spec_release 2
+%global spec_version 17.0.7.0.0.7
+%global spec_release 3
```
Comment (steelhead31) [inline linux/jre/suse/src/main/packaging/temurin/17/temurin-17-jre.spec:8]: should be spec\_release 1

File: linux/jre/suse/src/main/packaging/temurin/17/temurin-17-jre.spec [hunk 8]
```diff
@@ -176,6 +176,8 @@ fi
 %{prefix}
 
 %changelog
+* Wed Apr 26 2023 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 17.0.7.0.0.7-3.adopt0
+- Eclipse Temurin JRE 17.0.7+7 release 3.
```
Comment (steelhead31) [inline linux/jre/suse/src/main/packaging/temurin/17/temurin-17-jre.spec:180]: should be release 1.

**Review by steelhead31 — CHANGES_REQUESTED** [review]
Spec version should be reset.

**Review by steelhead31 — APPROVED** [review]
LGTM!

**Review by karianna — APPROVED** [review]

**Review by tellison — APPROVED** [review]

**Pull request merged**